e2e/test/scenarios/visualizations-tabular/visualizations-tabular-reproductions.cy.spec.js

```diff
@@ -950,3 +953,31 @@ describe.skip("issue 25415", () => {
     cy.findByTestId("TableInteractive-root").should("exist");
   });
 });
+
+describe("issue 45481", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+  });
+
+  it("should not crash when the table viz gets automatically pivoted (metabase#45481)", () => {
+    openOrdersTable();
+    openNotebook();
```

Comment:
Nit:
_Do not change this now to avoid another CI cycle!_

`openOrdersTable({ mode: "notebook" });`